CONTRIBUTING_A_NEW_QUEST.md

```diff
@@ -0,0 +1,390 @@
+For code style and more general info - see [CONTRIBUTING file](CONTRIBUTING.md#development.md).
+
+If you want to contribute code to StreetComplete, then making a new quest is one of the easiest programming tasks. Implementing a quest with the interface design matching an existing one is quite simple.
+
+Contributions like that are highly welcome and you would make mapping one more thing in OSM much easier!
+
+Reading the text below is not necessary to create a new quest - a basic method of duplicating existing ones and modifying them may be sufficient. And people were creating new quests before this documentation existed.
+
+But these materials may help or be quicker than trying to fully explore on your own how things work.
+
+# What is not required
+
+- expert programming ability - you do not need to be an expert programmer to make contributions, basic programming abilities are sufficient to implement a quest similar to existing ones
+- any Kotlin-specific or Android-specific knowledge - it is fine if you never did anything with Kotlin or Android
+- any GitHub specific knowledge
+
+Please [open an issue](https://github.com/streetcomplete/StreetComplete/issues/new) if you are stuck despite this. 
```

Comment:
I am not getting what is improved here.

Is it intended to clarify meaning? Fix some grammar issue? Or is it supposed to sound better?

"Regardless" seems wrong here - as invitation for doing this applies to people who passed minimal requirements (if someone never ever programmed they should not create an issue here)

Reply:
Yeah, clarify you're saying they got stuck despite the document, rather than the steps above.

> Or is it supposed to sound better?

Probably a bit of that.

> "Regardless" seems wrong here - as invitation for doing this applies to people who passed minimal requirements (if someone never ever programmed they should not create an issue here)

But you haven't listed any minimal requirements, just what they DON'T need!

Reply:
https://github.com/streetcomplete/StreetComplete/blob/e46a86b658ccfe6b20e73542d4033df611b04809/CONTRIBUTING_A_NEW_QUEST.md#required is section later, I need to think about it

Reply:
I fixed it by removing this segment after all.